frontend/src/models/Pressure.ts

```diff
@@ -0,0 +1,14 @@
+export enum PressureStatus {
+    Normal = 'Normal',
+    Critical = 'Critical',
+    Error = 'Error',
+}
+
+export class Pressure {
```

Comment:
If this is a class you could add physical property to hold mbar or bar? not sure

Reply:
Also, I don't find where this class is used. Why is it needed?